CMakeLists.txt

```diff
@@ -53,7 +53,7 @@ endif()
 
 ExternalProject_Add(tvm-build
   GIT_REPOSITORY https://github.com/apache/tvm
-  GIT_TAG v0.9.0
+  GIT_TAG da7b48f9487c9ee8eb8c6a6d7f80b59969f842c8 # v0.10.0
```

Comment:
CMake ExternalProject_add recommends using a specific git hash with
GIT_TAG because branches and tags can be updated to point to different
references.
https://github.com/ros2/rosbag2/pull/969

Reply:
Ack